src/meta/test/ProcessorTest.cpp

```diff
@@ -1452,7 +1454,7 @@ TEST(ProcessorTest, AlterTagTest) {
     cpp2::Schema changeSch;
     cpp2::ColumnDef column;
     column.name = "tag_0_col_10";
-    column.type.type_ref() = PropertyType::DOUBLE;
+    column.type.type_ref() = PropertyType::INT64;
```

Comment:
Note that the current column type of col_10 is int64. The failure should come from changing col with ttl. So I keep the column type the same, to avoid the error coming from illegal column type change.